pom.xml

```diff
@@ -76,7 +76,12 @@
         <xmlsec.version>2.0.8</xmlsec.version>
         <jts.version>1.12</jts.version>
         <jts.bundle.version>1.12_1</jts.bundle.version>
+
+        <!-- If you -->
         <karaf.version>4.1.1</karaf.version>
+        <felix.configadmin.version>1.8.8</felix.configadmin.version>
+        <!--TODO: Add our felix config admin version here-->
```

Comment:
:question: Should "felix.configadmin.version" above be 1.8.14 instead of 1.8.8?

Reply:
We had noticed that earlier. This property is only used in the itests. I just verified that updating it to `1.8.14` does not affect the itests, so it should be updated to `1.8.14` in this PR and reused.